form_utils/widgets.py

```diff
@@ -59,15 +61,65 @@ def render(self, name, value, attrs=None):
         return mark_safe(output)
 
 
+class ClearableFileInputRenderer(object):
```

Comment:
This is hijacking the way django 1.11 renders widgets and making it work with this lib. This code only runs in Django 1.11

Reply:
Can you elaborate more on what ensures this code only runs in django 1.11?

Reply:
https://github.com/PolicyStat/django-form-utils/pull/1/files#diff-5928fad501fed3b57076c4e760027bd3R137 This line of code is passing along an instance of this to the render method. It only gets passed along in Django 1.11 (the render method in Djagno 1.8 crashes if passed this instance.

Reply:
`django.VERSION > (1, 10, 0)` would allow django 1.10.x, with x > 0, right?

Reply:
Yes, but we are the only people that will ever use this lib and we will only ever be using it with django 1.8 and django 1.11. I specifically chose a number in the middle so I wouldn't have to worry figuring out if `>=` works when comparing tuples.